cpp/src/feather/metadata.cc

```diff
@@ -186,8 +184,9 @@ TableBuilder::TableBuilder(int64_t num_rows) {
   impl_.reset(new Impl(num_rows));
 }
 
-TableBuilder::TableBuilder() :
```

Comment:
gcc 4.4 doesn't have delegating constructors; we can just duplicate the body of the constructor it delegates to in this simple case.